src/test/java/com/atypon/wayf/integration/admin/AdminUserIntegrationTest.java

```diff
@@ -191,4 +195,19 @@ public void testGetMeInvalidToken() {
 
         userTestUtil.testMeInvalidToken(adminToken1, CURRENT_USER_INVALID_TOKEN_RESPONSE_JSON);
     }
+
+    @Test
+    public void testFilterPublishers() {
```

Comment:
For completeness we might want to pass in the admin view but without an admin token and assert a 401 response